scripts/dependencies/wix.ts

```diff
@@ -30,4 +32,32 @@ export class Wix extends GithubVersionGetter implements Dependency {
     await download(url, archivePath);
     await spawnFile('unzip', ['-o', archivePath, '-d', wixDir], { cwd: wixDir, stdio: 'inherit' });
   }
+
+  async getAvailableVersions(): Promise<string[]> {
+    return await getPublishedReleaseTagNames(this.githubOwner, this.githubRepo);
+  }
+
+  private wixVersionToSemver(version: string): string {
+    let onlyNumbers = version.replace(/^wix/, '').replace(/rtm$/, '');
```

Comment:
Alternatively, we can fetch the GH release (based on tag), and return the cleaned up release _name_.

Reply:
I'm going to leave it as it is for now, since it seems unlikely that there will be another release of Wix 3. But it is good to be aware of this as an option.